iam/api-client/service_accounts.py

```diff
@@ -73,27 +87,40 @@ def rename_service_account(email, new_display_name):
     """Changes a service account's display name."""
 
     # First, get a service account using List() or Get()
+    credentials = service_account.Credentials.from_service_account_file(
+        filename=os.environ['GOOGLE_APPLICATION_CREDENTIALS'],
+        scopes=['https://www.googleapis.com/auth/cloud-platform'])
+
+    service = googleapiclient.discovery.build(
+        'iam', 'v1', credentials=credentials)
+
     resource = 'projects/-/serviceAccounts/' + email
-    # pylint: disable=no-member
-    service_account = service.projects().serviceAccounts().get(
+
+    my_service_account = service.projects().serviceAccounts().get(
         name=resource).execute()
 
     # Then you can update the display name
-    service_account['displayName'] = new_display_name
-    service_account = service.projects().serviceAccounts().update(
+    my_service_account['displayName'] = new_display_name
+    my_service_account = service.projects().serviceAccounts().update(
         name=resource, body=service_account).execute()
 
     print('Updated display name for {} to: {}'.format(
-        service_account['email'], service_account['displayName']))
-    return service_account
+        service_account['email'], my_service_account['displayName']))
```

Comment:
Same problem here, need to rename service_account['email'] to my_service_account